### PR TITLE
[amberscript] Add CLEAR_COLOR parsing.

### LINKED ISSUE #344: Parse CLEAR_COLOR command

### PR DESCRIPTION
**Pull request by dj2**
This Cl adds CLEAR_COLOR parsing into the AmberScript parser. The EXPECT
EQ was also fixed to use the correct 0-255 range instead of 0.0-1.0
range for colour expectations.

Fixes #344.

### CODE REVIEW

File: src/amberscript/parser.cc [hunk 1]
```diff
@@ -1235,5 +1237,66 @@ Result Parser::ParseCopy() {
...
+    return Result("missing pipeline name for CLEAR_COLOR command");
+
+  auto* pipeline = script_->GetPipeline(token->AsString());
+  if (!pipeline) {
+    return Result("unknown pipeline for CLEAR_COLOR command: " +
+                  token->AsString());
+  }
+  if (!pipeline->IsGraphics()) {
+    return Result(
+        "CLEAR_COLOR command requires graphics pipeline, got compute");
```
Comment (hevrard) [inline src/amberscript/parser.cc]: This conditional + message make the assumption that a pipeline is either graphics or compute, which may not be future proof? That is nit-picking, but maybe the message should just say "CLEAR_COLOR command argument is not a graphics pipeline" ?

File: src/command.h [hunk 2]
```diff
@@ -418,6 +419,7 @@ class ClearColorCommand : public PipelineCommand {
   explicit ClearColorCommand(Pipeline* pipeline);
   ~ClearColorCommand() override;
 
+  // Colours are stored in the range 0.0-1.0
```
Comment (hevrard) [inline src/command.h]: Nit: `the range 0.0-1.0` -> `the range 0.0 - 1.0` (add spaces to exactly match the previous comment)

File: src/amberscript/parser.cc (hunk 1 above)
Comment (dj2) [reply src/amberscript/parser.cc]: Good point. Fixed other instances as well.

File: src/command.h (hunk 2 above)
Comment (dj2) [reply src/command.h]: done

**Pull request merged**